src/components/manage/Blocks/Grid/adapter.js

```diff
@@ -0,0 +1,19 @@
+export const GridBlockDataAdapter = ({
+  block,
+  data,
+  id,
+  onChangeBlock,
+  value,
+}) => {
+  let dataSaved = {
+    ...data,
+    [id]: value,
+  };
+  if (id === 'direction') {
+    dataSaved = {
+      ...dataSaved,
+      '@type': value,
```

Comment:
I don't understand this part; why does updating the direction set the type?

Reply:
Uh, this was a leftover from the column block experiments

Reply:
Removed.